frame/assistants/client-pangolin/src/client.rs

```diff
@@ -131,6 +132,15 @@ impl PangolinClient {
         }
     }
 
+    pub async fn subscribe_grandpa_justifications(&self) -> ClientResult<Subscription<sp_core::Bytes>> {
+        let sub = self.client.rpc().client.subscribe(
+            "grandpa_subscribeJustifications",
+            None,
+            "grandpa_unsubscribeJustifications",
+        ).await.unwrap();
```

Comment:
It's better to throw an error up than to panic. may you need add From trait for this error. like this
https://github.com/darwinia-network/bridger/blob/0529e26970066902120bead8060dbc1d528fa335/frame/assistants/client-pangolin/src/error.rs#L92-L96